source/includes/_fields_specification.md

```diff
@@ -0,0 +1,490 @@
+# Fields specification
+
+Dataset fields can be configured using a variety of options. These options include their type, their label, whether they are a filter or not, whether they should be discarded, or even how fields are order. These configuration options, collectively refered to as fields specification, are accessed using [processor-like objects](#the-processor-object).
+
+## Available configuration items
+
+Here is a comprehensive list of the available configuration items. Example usage for each one can be found in their dedicated section.
+
+Configuration name | Description
+------------------ | -----------
+[`rename`](#rename) | changes the name of the field and sets its label
+[`type`](#type) | sets the field type
+[`annotate`](#annotate) | annotates the field
+[`description`](#description) | describes the field
+[`order`](#order) | reorders fields
+[`delete`](#delete) | deletes a field
+
+## Rename
+
+> change the field named "original_field_id" to "new_field_id", and sets its label to "desired_label"
+
+```json
+{
+    "name": "rename",
+    "args": {
+        "from_name": "original_field_id",
+        "to_name": "new_field_id",
+        "label": "desired_label"
+    }
+}
+```
+
+Fields have a unique ID, as well as a human-friendly label, both of which are editable using this configuration item.
```

Comment:
Instead of "unique ID", we often refer to the name as "technical name".

Reply:
:+1: